change/@ni-nimble-tokens-7d1352e1-f82c-48b8-94fd-d86131297a6d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
Guess we can delete this file right? It's not the source of truth anymore?
https://github.com/ni/nimble/blob/cd78ec74ffdf3d47d405cbbf5cd8ecac6f34a988/packages/nimble-tokens/source/icons/Nimble_Iconography.ai

Reply:
Hmm, yeah, looks like we use a copy in an internal repo instead. Deleted it and updated docs that mention it.